tox.ini

```diff
@@ -83,7 +88,12 @@ deps =
 [testenv:pinned]
 deps =
     {[pinned]deps}
-    lxml==3.5.0
```

Comment:
Why was this removed?

Reply:
I was thinking of using the `testenv:pinned` part for replacing the `PyDispatcher` requirements for pinned environments other than pypy, (for `windows-pinned` and `asyncio-pinned`). I just remembered though that there is a test job in travis on this testenv. Let me try some other alternatives than making a mess.